examples/stable_fluid.py

```diff
@@ -8,11 +8,11 @@
 import numpy as np
 import time
 
-res = 600
+res = 512 #600
 dt = 0.03
-p_jacobi_iters = 30
+p_jacobi_iters = 100 #30
 f_strength = 10000.0
-curl_strength = 0
+curl_strength = 0 #3
```

Comment:
Just curious, is there a particular reason for these changes on the default parameters?

Reply:
Setting to 3, for example, make curls strong for visual enhancement, but not really physics-based.